Sources/Host/Linux/X86_64/PTraceX86_64.cpp

```diff
@@ -107,12 +109,24 @@ state32_to_user(user_fpregs_struct &user,
   //
   // SSE State
   //
-  user.mxcsr = state.sse.mxcsr;
-  user.mxcr_mask = state.sse.mxcsrmask;
-  auto xmm_space = reinterpret_cast<uint8_t *>(user.xmm_space);
+  xfpregs.fpregs.mxcsr = state.sse.mxcsr;
+  xfpregs.fpregs.mxcr_mask = state.sse.mxcsrmask;
+  auto xmm_space = reinterpret_cast<uint8_t *>(xfpregs.fpregs.xmm_space);
   for (size_t n = 0; n < array_sizeof(state.sse.regs); n++) {
     memcpy(xmm_space + n * 16, &state.sse.regs[n], sizeof(state.sse.regs[n]));
   }
+
+  //
+  //  EAVX State
+  //
+  auto ymmh = reinterpret_cast<uint8_t *>(xfpregs.ymmh);
+  for (size_t n = 0; n < array_sizeof(state.avx.regs); n++) {
+    size_t sseSize = sizeof(state.sse.regs[n]);
+    size_t avxSize = sizeof(state.avx.regs[n]);
+    auto avxHigh =
+        reinterpret_cast<const uint8_t *>(&state.avx.regs[n]) + sseSize;
+    memcpy(ymmh + n * 16, avxHigh, avxSize - sseSize);
```

Comment:
the size of one ymmh vector. I assume I could actually just use avxSize - sseSize again there.